build-in-container/README.md

```diff
@@ -0,0 +1,33 @@
+# Build-in-container
+The build-in-container tool provides the user with an easy-to-use, distribution and platform agnostic, container build for Mariner packages. The container is complete and independent, and eliminates the requirement for the user to interact with Mariner toolkit to build packages.
+
+Please install docker on your system before using the tool.
+
+## Usage
+The run.sh script presents these options
+-t      creates container image
+-b      creates container and builds Mariner
```

Comment:
A critica feature we need is the ability to build images as well as packages.

Reply:
building image will taken up in the next revision. Building packages in this revision